[2022-08-02 15:51:02,518] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: pokea.respond 2022-08-02T15:40:00+00:00 [queued]>
[2022-08-02 15:51:02,615] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: pokea.respond 2022-08-02T15:40:00+00:00 [queued]>
[2022-08-02 15:51:02,616] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 15:51:02,617] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-02 15:51:02,617] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 15:51:02,648] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-02T15:40:00+00:00
[2022-08-02 15:51:02,653] {standard_task_runner.py:51} INFO - Started process 13235 to run task
[2022-08-02 15:51:02,664] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'pokea', 'respond', '2022-08-02T15:40:00+00:00', '--job-id', '80', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pokea.py', '--cfg-path', '/tmp/tmpar4prd_a']
[2022-08-02 15:51:02,671] {standard_task_runner.py:76} INFO - Job 80: Subtask respond
[2022-08-02 15:51:02,849] {logging_mixin.py:103} INFO - Running <TaskInstance: pokea.respond 2022-08-02T15:40:00+00:00 [running]> on host b4dcda255553
[2022-08-02 15:51:03,131] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=pokea
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T15:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T15:40:00+00:00
[2022-08-02 15:51:03,134] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-02 15:51:03,267] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=pokea, task_id=respond, execution_date=20220802T154000, start_date=20220802T155102, end_date=20220802T155103
[2022-08-02 15:51:03,379] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-02 15:51:03,407] {local_task_job.py:118} INFO - Task exited with return code 0
